### PR TITLE
or-tools: 8.2

### PR DESCRIPTION
**Pull request by Mizux**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
no, did it from GitHub directly
ps: I'm the maintainer of google/or-tools 
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
NA
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
NA
-----

note: Since release v8.1 OR-Tools do not depends on Glog and GFlags anymore...

### CODE REVIEW

File: Formula/or-tools.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class OrTools < Formula
   desc "Google's Operations Research tools"
   homepage "https://developers.google.com/optimization/"
-  url "https://github.com/google/or-tools/archive/v8.1.tar.gz"
-  sha256 "beb9fe379977033151045d0815d26c628ad99d74d68b9f3b707578492723731e"
+  url "https://github.com/google/or-tools/archive/v8.2.tar.gz"
+  sha256 "cf40715fa5cfeee88e2c8f5583465182c8dedf60b4eb7c4a967b32ff61ac4302"
   license "Apache-2.0"
   revision 1
```
Comment (carlocab) [inline Formula/or-tools.rb:7]:
```suggestion
```
The `revision` lines need to be removed at version bumps.

**Review by carlocab — CHANGES_REQUESTED** [review]
Thanks for your PR, @Mizux.

**Comment by carlocab** [comment]
Also, would you know if this builds on ARM yet?

**Comment by Mizux** [comment]
For ARM build we manage to build or-tools using linaro toolchain
see: https://github.com/google/or-tools/runs/2030251854?check_suite_focus=true

I don't know the process how to build for homebrew-core arm BUT code should be ready !

**Comment by Mizux** [comment]
Fixup the revision line and amend the commit message.

**Comment by carlocab** [comment]
Hmm, ok... well, let's attempt an ARM build and see what it spits out. If it builds, then great. If not, then maybe the error message tells us something useful.

**Comment by carlocab** [comment]
The build succeeded on ARM but the test failed:
```
==> /usr/bin/clang++ -std=c++17 -I/opt/homebrew/Cellar/or-tools/8.2/include -L/opt/homebrew/Cellar/or-tools/8.2/lib -lortools -L/opt/homebrew/opt/gflags/lib -lgflags -L/opt/homebrew/opt/glog/lib -lglog -L/opt/homebrew/opt/abseil/lib -labsl_time /opt/homebrew/Cellar/or-tools/8.2/share/or-tools/simple_lp_program.cc -o simple_lp_program
ld: warning: directory not found for option '-L/opt/homebrew/opt/gflags/lib'
ld: warning: directory not found for option '-L/opt/homebrew/opt/glog/lib'
ld: library not found for -lgflags
```
We should be seeing these errors on the Intel nodes shortly.
The problem comes from these lines:

https://github.com/Homebrew/homebrew-core/blob/fa62c2f9815ca5d77a8601e793a6ecaa5dadac67/Formula/or-tools.rb#L45-L64

**Comment by Mizux** [comment]
oups forget to fixup please wait my push -f

**Comment by Mizux** [comment]
done !

**Comment by Mizux** [comment]
One more question, i'm following https://github.com/Homebrew/homebrew-core/runs/2031294839?check_suite_focus=true but i was not able to see the compilation trace etc
@carlocab do you know where I can find it ?

**Comment by SMillerDev** [comment]
You can't unless it fails

**Review by carlocab — APPROVED** [review]
Thanks, @Mizux.

**Pull request closed without merging**